source/includes/entityref/_candidateshiftpreference.md

```diff
@@ -0,0 +1,73 @@
+# CandidateShiftPreference
+
+CandidatePreferences is used for storing a candidate's preferences for times and days when they want to work in association with the shiftTypes that are configured.
```

Comment:
Change to something like:
Stores a candidate's preferences for times and days to work in association with the shiftTypes that are configured.

If ShiftType is an entity, use a capital S on it.